Readme.md

```diff
@@ -3,81 +3,85 @@
 
 This will install and run all services required to run all features in Pinpoint in docker containers joined with same network.
- - Pinpoint-Web Server
- - Pinpoint-Collector
- - Pinpoint-Agent(ready to be used)
- - Pinpoint-Flink
- - Pinpoint-Zookeeper
- - Pinpoint-Hbase
- - Pinpoint-QuickStart(a sample application)  
+
+- pinpoint-web
+- pinpoint-collector
+- pinpoint-agent(ready to be used)
+- pinpoint-flink
+- pinpoint-zookeeper
+- pinpoint-hbase
+- pinpoint-quickstart(a sample application)  
+
 This may take several minutes to download all necessary images.
 
 You can replace `QuickStart` application part with your application to start monitoring.  
- - check [`Testing QuickStart application`](#testing-quickstart-application) for a quick demo of pinpoint
- - check [`Monitoring YOUR Application`](#monitoring-your-application) part for further details
+
+- check [`Testing QuickStart application`](#testing-quickstart-application) for a quick demo of pinpoint
+- check [`Monitoring YOUR Application`](#monitoring-your-application) part for further details
 
 ### Flink configuration (optional)
 
 After all containers are started and ready to go. There is one more thing to do to use all existing features in Pinpoint.
 It's not mandatory, but to use all the features and since it's a simple task, let's take care of it.
 
 Register a `job` on to pinpoint-flink server.  
-You can build the `job` from the [open-source of Pinpoint](https://github.com/naver/pinpoint), additional guide is [here](https://github.com/naver/pinpoint/blob/master/doc/application-inspector.md#application-inspector).   
+You can build the `job` from the [open-source of Pinpoint](https://github.com/naver/pinpoint), additional guide is [here](https://github.com/naver/pinpoint/blob/master/doc/application-inspector.md#application-inspector).
 or  
 you can simply upload pre-built jar file under pinpoint-flink/build/pinpoint-flink-job-{pinpoint-version}.jar (beware of the version, it should matched with *PINPOINT_VERSION* in .env file)  
 If anyone have solution to put the job file into flink image without doing manually, please let us know.
 
-Pinpoint-Flink server is running on [port 8081](http://localhost:8081/#/submit). From `submit new job` menu
+pinpoint-flink server is running on [port 8081](http://localhost:8081/#/submit). From `submit new job` menu
 Submit the jar file with *com.navercorp.pinpoint.flink.StatStreamingVer2Job* in entry class as below image.
 
-![Pinpoint](https://github.com/naver/pinpoint-docker/blob/master/docs/Pinpoint-Flink%20upload.png)
- 
+![Pinpoint](https://github.com/naver/pinpoint-docker/blob/master/docs/pinpoint-flink%20upload.png)
+
 ### Testing QuickStart application
- 
-Now you are ready to monitor the sample application(Pinpoint-QuickStart [port 8000](http://localhost:8000)) provided.
+
+Now you are ready to monitor the sample application(pinpoint-QuickStart [port 8000](http://localhost:8000)) provided.
```

Comment:
I think it will be better to leave all the titles as Capital.

not only the ones I've commented, but all of the titles.

Reply:
I have reset them all and just change the command line which cannot exec.
Even though, I though that we should let all the names equal to the directory/project names.